requirements.txt

```diff
@@ -0,0 +1,50 @@
+# All the requirements needed for the project + running tests
```

Comment:
Please do not add any more changes to this pr.
So no need to add requirements.txt